test/Mono.Linker.Tests.Cases/Reflection/MemberUsedViaReflection.cs

```diff
@@ -0,0 +1,487 @@
+﻿using System;
+using System.Diagnostics.CodeAnalysis;
+using System.Reflection;
+using Mono.Linker.Tests.Cases.Expectations.Assertions;
+using Mono.Linker.Tests.Cases.Expectations.Metadata;
+
+namespace Mono.Linker.Tests.Cases.Reflection
+{
+	[SetupCSharpCompilerToUse ("csc")]
+	public class MemberUsedViaReflection
+	{
+		public static void Main ()
+		{
+			// Normally calls to GetMember use regex to find values, we took a conservative approach
+			// and preserve not based on the string passed but on the binding flags requirements
```

Comment:
I don't get this - regex?
I'm fine if we don't implement the name based lookups (this is not very common), but the comment here is weird.

Reply:
I see - well I would not call "C*" a regex (it's actually not a valid regex) - more like "wildcard".
Can you please rename anything with "Regex" in this class - it doesn't matter to what really, but regex is confusing.